dev/local-environment-dynamic/modules/partner-chains-setup.txt

```diff
@@ -1,6 +1,7 @@
   partner-chains-setup:
     container_name: partner-chains-setup
     image: ${PARTNER_CHAINS_NODE_IMAGE}
+    pull_policy: always
```

Comment:
I'm running `setup.sh -i partner-chains-node:latest` and then I get:
```
Error response from daemon: pull access denied for partner-chains-node, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```
, so it is also problematic with our node image when developing locally